packages/builder/friendly-errors-webpack-plugin/README.md

```diff
@@ -0,0 +1,162 @@
+# Friendly-errors-webpack-plugin
```

Comment:
We should rewrite the README because it does not match the current implementation.